ipam/pool.go

```diff
@@ -65,17 +65,21 @@ type addressPool struct {
 
 // AddressPoolInfo contains information about an address pool.
 type AddressPoolInfo struct {
-	Subnet     net.IPNet
-	Gateway    net.IP
-	DnsServers []net.IP
-	IsIPv6     bool
+	Subnet        net.IPNet
+	Gateway       net.IP
+	DnsServers    []net.IP
+	IsIPv6        bool
+	Capacity      int
+	Available     int
+	UnhealthyAddr []string
```

Comment:
Let's name this UnhealthyAddr**s** since this is an array, not a single address. Also addresses should be of type net.IP.